tap_facebook/__init__.py

```diff
@@ -153,6 +153,8 @@ def should_retry_api_error(exception):
             return True
         elif isinstance(exception, TypeError) and str(exception) == "string indices must be integers":
             return True
+        elif isinstance(exception, AttributeError):
```

Comment:
We can add it in the or of `if isinstance(exception, FacebookBadObjectError)`, instead of adding a new elif.

Reply:
Updates as per suggestion.